contracts/deploy/mainnet/095_ogn_buyback.js

```diff
@@ -0,0 +1,76 @@
+const { isFork } = require("../../test/helpers");
+const addresses = require("../../utils/addresses");
+const {
+  deploymentWithGovernanceProposal,
+  deployWithConfirmation,
+} = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "095_ogn_buyback",
+    // forceSkip: true,
+    // onlyOnFork: true, // this is only executed in forked environment
+    reduceQueueTime: true, // just to solve the issue of later active proposals failing
+    proposalId: "",
+  },
+  async ({ ethers }) => {
+    const cOETHBuybackProxy = await ethers.getContract("OETHBuybackProxy");
+    const cOUSDBuybackProxy = await ethers.getContract("BuybackProxy");
+
+    const cSwapper = await ethers.getContract("Swapper1InchV5");
+
+    // Deploy new OETHBuyback implementation
+    const dOETHBuybackImpl = await deployWithConfirmation(
```

Comment:
nit: in the `deployActions.js` you could create an `upgradeBuyback` function that can be reused in the future deploy scripts. But it is also fine as it is